aptos-move/framework/aptos-framework/sources/resource_account.spec.move

```diff
@@ -29,9 +77,26 @@ spec aptos_framework::resource_account {
         metadata_serialized: vector<u8>,
         code: vector<vector<u8>>,
     ) {
-        // TODO: Calls `code::publish_package_txn`.
-        // TODO: Could not verify `code::publish_package_txn` because can't get `resource` and `resource_signer_cap`.
         pragma verify = false;
```

Comment:
Why is this not verified?

Reply:
It is related to a vector enum loop in code.move, which we can't verify yet

Reply:
by the way, if we used aborts_if_is_partial, it will throw an boogie error:  Error: wrong number of result variables in call to $1_vector_length'$1_code_PackageMetadata':

Reply:
@UIZorrot Could you file an issue for this internal boogie error? Thanks